CHANGELOG.md

```diff
@@ -23,13 +25,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Parameters
 
-| Old parameter | New parameter         |
-| ------------- | --------------------- |
-|               | mbuffer_mem           |
-|               | samtools_sort_threads |
-|               | skip_repeat_analysis  |
-|               | skip_snv_calling      |
-|               | skip_sv_calling       |
+| Old parameter   | New parameter            |
+| --------------- | ------------------------ |
+|                 | mbuffer_mem              |
+|                 | samtools_sort_threads    |
+|                 | skip_repeat_analysis     |
```

Comment:
I think `skip_repeat_calling` would be more inline with the subworkflow name, the current output and usage docs ("Variant calling - repeat expansions"), and the parameters below.

Reply:
hmmm.. good point. The workflow includes stranger which, as you know already, is used to annotate STRs. So it does perform more than repeat calling. Perhaps I need to change the name of the subworkflow, and its references in the pipeline 😅

Reply:
I agree skip_repeat_analysis is more inline with what it actually does. 

What about splitting the subworkflow into call and annotate, like you do for SNVs and SVs? Then we could use the same annotation subworkflow in both raredisease and Nallo, and you can have `skip_repeat_calling` and `skip_repeat_annotation`.

Reply:
Great minds think like 😆 
That's what I am actually doing right now :D Its much easier, and like you said, it is in line with what we are doing for other variant types.

Reply:
Very nice! ⭐